crates/bevy_pbr/src/render/light.rs

```diff
@@ -1210,16 +1210,18 @@ pub fn prepare_lights(
                 .create_view(&TextureViewDescriptor {
                     label: Some("point_light_shadow_map_array_texture_view"),
                     format: None,
-                    #[cfg(any(
-                        not(feature = "webgl"),
-                        not(target_arch = "wasm32"),
-                        feature = "webgpu"
+                    #[cfg(all(
+                        not(ios_simulator),
```

Comment:
A comment on why this is necessary should be added here in case this changes in the future.

Reply:
Yeah, hopefully Apple eventually adds support for this, so good idea to add a comment. Fixed!